blockchain/process.go

```diff
@@ -172,7 +172,7 @@ func (b *BlockChain) ProcessBlockHeader(header *wire.BlockHeader, flags Behavior
 	// positional checks, and create a block index entry for it.
 	b.chainLock.Lock()
 	const checkHeaderSanity = true
-	_, err := b.maybeAcceptBlockHeader(header, flags, checkHeaderSanity)
+	_, err := b.maybeAcceptBlockHeader(header, BFNone, checkHeaderSanity)
```

Comment:
It looks like this is always none as well and could be removed from the `b.maybeAcceptBlockHeader(header, BFNone, checkHeaderSanity)` signature. Maybe that's coming later?